src/main/java/org/candlepin/policy/js/compliance/hash/HashableStringGenerators.java

```diff
@@ -204,6 +205,15 @@ public String generate(Entry<String, String> target) {
      */
     private static class ConsumerGenerator implements HashableStringGenerator<Consumer> {
 
+        private static final Set<String> RELEVANT_FACTS = Set.of(
```

Comment:
Even though for now it's only the Hasher that cares about these, conceptually they are part of the business logic (they belong to the compliance policy). So instead of hiding them here, let's move move them into a proper Enum called something like `ComplianceStatusFacts` (similar to `CloudProfileFacts`), placed in org.candlepin.policy.js.compliance package.

Reply:
Done.